time-elements.js

```diff
@@ -163,6 +163,27 @@
     }
   };
 
+  RelativeTime.prototype.microTimeAgo = function() {
+    var ms = new Date().getTime() - this.date.getTime();
+    var sec = ms / 1000;
+    var min = sec / 60;
+    var hr = min / 60;
+    var day = hr / 24;
+    var month = day / 30;
+    var year = month / 12;
+    if (min < 1) {
+      return '1m';
+    } else if (min < 60) {
+      return Math.round(min) + 'm';
```

Comment:
@dgraham do you remember why this rounds after comparison, not before? The other non-micro checks round out all the values first.